src/mocsolver.jl

```diff
@@ -59,11 +59,17 @@ function solve(
     prob::MoCProblem;
     max_iterations::Int=1000, max_residual::Real=1e-7, debug::Bool=false
 )
-    return _solve_eigenvalue_problem(prob, max_iterations, max_residual, debug)
+    return _solve(prob, max_iterations, max_residual, debug)
 end
 
-function _solve_eigenvalue_problem(prob::MoCProblem, max_iter::Int, max_ϵ::Real, debug::Bool)
+function solve(
+    prob::MoCProblem, fixed_sources::Matrix{Float64};
```

Comment:
it is not ok to use `Float64` here. Notice that we have never specified the problem element type.